server/routes/induction/update/affectAbilityToWorkUpdateController.ts

```diff
@@ -18,12 +20,15 @@ export default class AffectAbilityToWorkUpdateController extends AffectAbilityTo
 
   getBackLinkUrl(req: Request): string {
     const { prisonNumber } = req.params
+    const { pageFlowHistory } = req.session
```

Comment:
Most of the changes to "update" controller themselves follow a similar pattern where I've:

Updated the methods that get the back link URL and aria text to make sure it uses the page flow history if there is one

Implement the code in the submit handler to detect if this is part of an update Question Set journey (updateInductionQuestionSet is on the session) and redirect to the next screen.
Most of the redirects are simple like this one as the next page is "known" - there are a couple of exceptions where the next page is dynamically worked out based on data in the Induction